main.go

```diff
@@ -537,12 +526,26 @@ func main() {
 	// 	log.Fatal(err)
 	// }
 
-	if err := runner.Run(ctx, conn, lastRanMigrationID); err != nil {
-		log.Fatal(err)
-	}
-
+	var org string = "default"
 	app := &cli.App{
 		Name: "store",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "org",
+				Value:       "default",
+				Usage:       "org to connect to",
+				Destination: &org,
+			},
+		},
+		Action: func(cCtx *cli.Context) error {
+			db, err := connectDB(org)
+			if err != nil {
+				log.Fatal("failed to open database:", err)
```

Comment:
make sure to return errors as much as possible:
```suggestion
				return err
```